bundles/org.eclipse.jface/src/org/eclipse/jface/viewers/FocusCellOwnerDrawHighlighter.java

```diff
@@ -106,7 +106,7 @@ private void markFocusedCell(Event event, ViewerCell cell) {
 
 	private void removeSelectionInformation(Event event, ViewerCell cell) {
 		GC gc = event.gc;
-		if ("gtk".equals(SWT.getPlatform())) { //$NON-NLS-1$
+		if(Util.isGtk()){
```

Comment:
Please leave a space after the `if` and before the `{`.

Reply:
done the changes.